docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -990,10 +991,11 @@ Request:
 --------------------------------------------------
 PUT /_enrich/policy/my-policy
 {
-    "type": "exact_match",
-    "indices": "users",
-    "match_field": "email",
-    "enrich_fields": ["first_name", "last_name", "address", "city", "zip", "state"]
+    "exact_match": {
+        "indices": "users",
+        "match_field": "email",
+        "enrich_fields": ["first_name", "last_name", "address", "city", "zip", "state"]
+    }
```

Comment:
Do we have an example to includes the `query` ?  (its fine if it is not part of the PR, but probably want it in there somewhere)

Reply:
No, there is not. It makes sense to emphasise that in a special section with the fact that reference data can be read from multiple indices.